hi-ml-azure/src/health_azure/himl.py

```diff
@@ -539,16 +539,20 @@ def submit_run_v2(workspace: Optional[Workspace],
 
     # number of nodes and processes per node cannot be less than one
     num_nodes = num_nodes if num_nodes >= 1 else 1
-    processes_per_node = processes_per_node if processes_per_node >= 1 else 1
-
-    if hyperparam_args:
-        param_sampling = hyperparam_args[PARAM_SAMPLING_ARG]
-
-        for sample_param, choices in param_sampling.items():
-            input_datasets_v2[sample_param] = choices.values[0]
-            cmd += f" --{sample_param}=" + "${{inputs." + sample_param + "}}"
+    if pytorch_processes_per_node is not None:
+        pytorch_processes_per_node = pytorch_processes_per_node if pytorch_processes_per_node >= 1 else 1
```

Comment:
Is this in case a value <= 0 is provided? If so I think it might be better to raise an error / include an assert here instead.

Reply:
I've cleaned that up, disallowing all those wrong values.